docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -125,55 +138,21 @@ For multi-platform communication over federation, we need the connecting instanc
+should be ready to accept/reject such a request through VOLTTRON's admin web interface. To facilitate this process, the
+VOLTTRON platform exposes a web-based server API for requesting, listing, approving, and denying certificate requests. For
+more detailed description, refer to :ref:`Agent communication to Remote RabbitMQ instance <Agent-Communication-to-Remote-RabbitMQAfter the CSR request is accepted, an authenticated shovel/federation connection can be established.
 
-Agents trying to connect to a remote instance directly need to have a public certificate signed by the remote instance for
-authenticated SSL-based connection.  To facilitate this process, the VOLTTRON platform exposes a web-based server API
-for requesting, listing, approving, and denying certificate requests.  For more detailed description, refer to
-:ref:`Agent communication to Remote RabbitMQ instance <Agent-Communication-to-Remote-RabbitMQ>`
 
 Installation Steps
 ------------------
 
 1. Setup two VOLTTRON instances using the instructions at :ref:`platform installation steps for RMQ <RabbitMQ-Install>`.
 **Please note that each instance should have a unique instance name and should be running on a machine/VM that has a unique host name.**
 
-2. In a multi-platform setup that needs each platform to communicate with each other using RabbitMQ over SSL,
-each VOLTTRON instance should should trust the ROOT CA of the other instance (RabbitMQ root ca). To achieve this, do the following:
-
-   a. Transfer (scp/sftp/similar) voltttron_home/certificates/certs/<instance_name>-root-ca.crt to a temporary
-      location on the other volttron instance machine. For example, if you have two instances called v1 and v2,
-      use scp to copy and transfer v1's 'v1-root-ca.crt' to v2 and v2's 'v2-root-ca.crt' to v1.
-
-       Note: If using virtual machines (VM) to scp files between VM's, openssh should be installed and running.
-
-   b. Append the contents of the transferred root ca to the instance's trusted-cas.crt file. Do this on both the instances. Now both
-      the instances <instance_name>-trusted-cas.crt will have two certificates.
-
-      For example:
-
-      On v1:
-      cat /tmp/v2-root-ca.crt >> VOLTTRON_HOME/certificates/certs/v1-trusted-cas.crt
-
-      On v2:
-      cat /tmp/v1-root-ca.crt >> VOLTTRON_HOME/certificates/certs/v2-trusted-cas.crt
-
-3. `Note: This step is required only when you update the root certificate and not
-required when you add a new shovel/federation between the same hosts.`
-Stop volttron, stop rabbitmq server, and start volttron on both the
-instances:
-
-.. code-block:: bash
-
-    ./stop-volttron
-    ./stop-rabbitmq
-    ./start-volttron
-
-4. Identify upstream servers (publisher nodes) and downstream servers
+2. Identify upstream servers (publisher nodes) and downstream servers
 (collector nodes). To create a RabbitMQ federation, we have to configure
 upstream servers on the downstream server and make the VOLTTRON exchange
 "federated".
```

Comment:
This statement is confusing. Can this be reworded to something like " To create a RabbitMQ federation, we have to configure the downstream volttron instance to create federated exchange. A federated exchange links to other exchanges. In this case, the downstream federated exchange links to the upstream exchange. Conceptually, messages published to the upstream exchanges are copied to the federated exchange, as though they were published directly to the federated exchange." - The text is based on documentation on RabbitMQ site.

Reply:
I would rather not. Too much detail, users will get lost. There is separate documentation to understand federation. I can point to that, if needed.

Reply:
Addressed